events_journal/src/test/java/com/adobe/event/journal/JournalServiceTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.event.journal;
+
+import com.adobe.ims.JWTAuthInterceptor;
+import com.adobe.ims.util.TestUtil;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import feign.RequestInterceptor;
+import org.junit.Rule;
+import org.junit.Test;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static junit.framework.TestCase.assertNotNull;
+
+public class JournalServiceTest {
+
+  private static final String POSITION = "10";
+
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(9999);
+
+  @Test
+  public void journalServiceTest(){
+
+    assertNotNull(getJournalService());
+
+//    /** get */
+    stubFor(get(anyUrl())
+            .withHeader(TestUtil.IMS_ORG_ID_HEADER, equalTo(TestUtil.IMS_ORG_ID))
+            .willReturn(aResponse()
+                    .withStatus(200)
+            ));
+
+    getJournalService().get(TestUtil.getWorkspace().getImsUrl());
+    verify(getRequestedFor(anyUrl()));
```

Comment:
please enrich the sub with populated json response so you could assert the resulting `JournalEntry` object